microsoftgraph/microsoft-graph-explorer-v4
# Task: Add logging for script errors

## Overview
Adds telemetry for #2344 


### Notes
This PR adds telemetry for all script errors. We will be able to tell where exactly the errors are coming from. 

## Testing Instructions

* How to test this PR
1. On the .tsx file, add:
![image](https://user-images.githubusercontent.com/45680252/214238911-c8eac106-0c99-4eab-8d5f-d1ea30c46800.png)
2. Now open the network tab and see the errors tracked. Notice that the exception data contains a source object that shows where the error is originating from:
3. 
![image](https://user-images.githubusercontent.com/45680252/214239138-e3bffb54-1e83-42aa-bd57-30c6218b69db.png)
